runner/runner.py

```diff
@@ -96,7 +98,7 @@ def train(self):
 
     def evaluate(self, algorithm, checkpoint_path: str = None, seed: int = 42):
         # Create an agent to handle the environment
-        agent = algorithm(config=self.config)
+        agent = algorithm
```

Comment:
How can we justify omitting the configuration parameter in the transition from `agent = algorithm(config=self.config)` to `agent = algorithm`? Might there still be advantages to retaining this type of implementation?